### PR TITLE
Lesson_page

### PR DESCRIPTION
**Pull request by yongsokheng**

### CODE REVIEW

File: Gemfile [hunk 1]
```diff
@@ -34,6 +34,7 @@ gem "sdoc", "~> 0.4.0", group: :doc
 gem "bootstrap-sass", "3.2.0.0"
 gem "will_paginate", "3.0.7"
 gem "bootstrap-will_paginate", "0.0.10"
+gem "faker", "1.4.2"
```
Comment (RathanakSreang) [inline Gemfile]: Put in develop

File: app/assets/javascripts/application.js [hunk 2]
```diff
@@ -15,3 +15,23 @@
 //= require turbolinks
 //= require bootstrap
 //= require_tree .
+
+$(document).ready(function(){
+  $('.word-list').hide();
```
Comment (RathanakSreang) [inline app/assets/javascripts/application.js]: `'`  to `"`

File: app/views/lessons/show.html.erb [hunk 3]
```diff
@@ -0,0 +1,21 @@
+<div class="row">
+  <div class="col-md-8">
+    <div class="title">
+      <h2><%= @lesson.category.name %></h2>
+      <h4 class="count alert alert-info"></h4>
+    </div>
+    <%= form_for [@lesson.category, @lesson] do |f| %>
+        <%= f.fields_for :word_lessons do |ff| %>
```
Comment (dieunb) [inline app/views/lessons/show.html.erb]: indent

File: app/views/lessons/show.html.erb [hunk 4]
```diff
@@ -0,0 +1,21 @@
...
+        <div class="word-list">
```
Comment (dieunb) [inline app/views/lessons/show.html.erb]: indent

**Pull request merged**